homework/max-of-vector/maxOfVector.hpp

```diff
@@ -3,6 +3,12 @@
 #include <limits>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
+    
+    int maximal = vec[0];
+
+    for(auto& element : vec )
+    if(element > maximal){
+    maximal = element;
+    }
+    return maximal;
```

Comment:
Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..